internal/rcm/api.go

```diff
@@ -81,7 +81,7 @@ func (api *API) submit(writer http.ResponseWriter, request *http.Request, _ http
 
 	type Repository struct {
 		URL      string `json:"url"`
-		Checksum string `json:"checksum"`
+		Checksum string `json:"checksum,omitempty"`
```

Comment:
My thinking was: Make it possible to use RCM API at least with it (because the codebase currently require it), but sure it makes sense to remove it completely.